types/aws-lambda/trigger/api-gateway-authorizer.d.ts

```diff
@@ -221,7 +221,7 @@ export interface Condition {
 export type Statement = BaseStatement & StatementAction & (StatementResource | StatementPrincipal);
 
 export interface BaseStatement {
-    Effect: string;
+    Effect: "Allow" | "Deny";
```

Comment:
Could you pull this out to a `export type StatementEffect =`?

Reply:
Added in [1825bbe](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/69715/commits/1825bbefd3023f44a41b083f5716ab14dbf4f3ee)